### PR TITLE
[Sharing] Permissions now in dropdown menu

### PR DESCRIPTION
**Pull request by rullzer**
As discusses with jan.

* Detailed permissions displayed in qtoolboxmenu
* Made share rows slightly smaller

Bug fix:

* Do not show delete permissions for file shares

### CODE REVIEW

**Comment by rullzer** [comment]
@jancborchardt this is how it looks now:

new

File: src/gui/shareusergroupwidget.cpp [hunk 1]
```diff
@@ -196,27 +198,39 @@ void ShareUserGroupWidget::slotCompleterActivated(const QModelIndex & index)
...
-  _showDetailedPermissions(false)
+  _isFile(isFile)
 {
     _ui->setupUi(this);
 
     _ui->sharedWith->setText(share->getShareWith()->format());
+ 
+    // Create detailed permissions menu
+    QMenu *menu = new QMenu(this);
+    _permissionCreate = new QAction(tr("create"), this);
```
Comment (ckamm) [inline src/gui/shareusergroupwidget.cpp]: isn't create also only relevant for !isFile?

**Comment by ckamm** [comment]
Rest looks good!

**Comment by dragotin** [comment]
:+1:

**Comment by jancborchardt** [comment]
@rullzer awesome! Just one thing – similar to the delete icon, use the `more` icon for that button: https://github.com/owncloud/core/blob/master/core/img/actions/more.png
(not just the dots and neither the triangle)

File: src/gui/shareusergroupwidget.cpp (hunk 1 above)
Comment (rullzer) [inline src/gui/shareusergroupwidget.cpp]: create is also overwrite I think... (I just followed the webinterface).

**Comment by rullzer** [comment]
@jancborchardt I now use the more icon. Howver I have no idea how to remove the tiny arrow...

**Comment by rullzer** [comment]
I'll merge this and somebody with more styling fu shall fix the tiny triangle...

**Pull request merged**